awslimitchecker/runner.py

```diff
@@ -358,6 +361,11 @@ def console_entry_point(self):
             ta_refresh_timeout=args.ta_refresh_timeout
         )
 
+        if args.skip_service is not None:
+            if args.service is not None:
+                raise SystemExit(0)
```

Comment:
Aside from this one last bit here, your last commit looks good; I'll merge once these conditionals are removed and tests pass.